proto/eth/v1/generated.ssz.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by fastssz. DO NOT EDIT.
-// Hash: ccd00c4ea3e7548025f7d1d122d737f44048e5d147a6a95fc109138ad510329b
+// Hash: 2ef8662cd6bab2d476ca79b4a918b5f0177316ea961f47983c0976eb415e46aa
```

Comment:
Curious... what is this hash and why was it the only thing changed in this file?

Reply:
I'm not quite sure what the purpose of this hash is. Looking at the fastssz source code it appears to concatenate the source files in the source package together, then computes a sha256 on the resulting blob. But it does so by iterating over a map, so presumably this changes every time we regenerate ssz: https://github.com/ferranbt/fastssz/blob/master/sszgen/main.go#L340